### PR TITLE
Future nextweb

### PR DESCRIPTION
**Pull request by tnyandoro**
Please review my work for responsive design on The Next Web clone. The site was done in html, css flex, css grid and some media queries to make it responsive

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,12 +4,1019 @@
...
+  <link rel="stylesheet" href="https://kit-free.fontawesome.com/releases/latest/css/free-v4-shims.min.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Nova+Square&display=swap">
+  <link href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/5b1eeceadd.js" crossorigin="anonymous"></script>
 </head>
 
 <body>
-  <h1>Hello world!</h1>
-</body>
+  <!-- navigation -->
```
Comment (arslanbisharat) [inline index.html:18]: remove comments that are not necessary

File: index.html [hunk 2]
```diff
@@ -4,12 +4,1019 @@
...
+      <div class="rightnav center-flex">
+        <i class="fas fa-search searchbtn"></i>
+        <i class="hamburger fas fa-bars" id="bar"></i>
+        <img src="images/event.png" alt="eventlogo" class="hiddenmd event-logo" />
+        <span class="time hiddenmd font-bold">17:18:35:51</span>
+        <button class="ticket hiddenmd font-bold" type="button">GET TICKETS</button>
+      </div>
+    </nav>
+
+    <!--main navigation for all screen -->
```
Comment (arslanbisharat) [inline index.html:72]: remove comments that are not necessary

File: index.html [hunk 3]
```diff
@@ -4,12 +4,1019 @@
...
+              EverC Raises $35M in Series B Funding) on FinSMEs
+            </p>
+          </a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  
+  <!-- this is the fourth section with grid -->
```
Comment (arslanbisharat) [inline index.html:364]: remove comments that are not necessary

File: index.html [hunk 4]
```diff
@@ -4,12 +4,1019 @@
...
+  <section class="section-4 d-sm-block d-xsm-block">
+    <!-- section for single item with flex -->
```
Comment (arslanbisharat) [inline index.html:366]: remove comments that are not necessary

File: index.html [hunk 5]
```diff
@@ -4,12 +4,1019 @@
...
+        <a href="#" class="textstyle-plain">
+          <p class="pic-headline mainfont font-bold">
+            Remembering Raven Thorogood III: The chimp that smoked Wall Street
+          </p>
+        </a>
+        <p class="view-history bk mainfont">David Canellis-5 days ago</p>
+      </div>
+
+      
+      <!-- second div with flex direction -->
```
Comment (arslanbisharat) [inline index.html:382]: remove comments that are not necessary

File: index.html [hunk 6]
```diff
@@ -4,12 +4,1019 @@
...
+          </li>
+          <li>
+            Satoshi Nakaboto: ‘Researchers find new way to launder money with Bitcoin’
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    
+    <!-- section for single item with flex -->
```
Comment (arslanbisharat) [inline index.html:419]: remove comments that are not necessary

File: index.html [hunk 7]
```diff
@@ -4,12 +4,1019 @@
...
+    <div class="section-4-item tn-flex">
+      <h2 class="secondaryfont">TECH <i class="fas fa-chevron-right"></i></h2>
+      <!-- Main pic div -->
```
Comment (arslanbisharat) [inline index.html:422]: remove comments that are not necessary

File: index.html [hunk 8]
```diff
@@ -4,12 +4,1019 @@
...
+      <div class="sec-4-big-pic-second">
+        <p class="headline-header bk mainfont">MOST POPULAR</p>
+        <a href="#" class="textstyle-plain">
+          <p class="pic-headline mainfont font-bold">
+            Warzone is infinitely more fun without vehicles (even though it's just a bug)
+          </p>
+        </a>
+        <p class="view-history bk mainfont">Yaron Yitzhak  19 hours ago</p>
+      </div>
+      <!-- second div with flex direction -->
```
Comment (arslanbisharat) [inline index.html:432]: remove comments that are not necessary

File: index.html [hunk 9]
```diff
@@ -4,12 +4,1019 @@
...
+          <p class="flexed-header mainfont">Latest</p>
+          <a href="#" class="textstyle-plain">
+            <p class="mainfont">
+              <strong>How to make collaborative playlists on Spotify and YouTube Music’</strong>
+            </p>
+          </a>
+        </div>
+        <div class="small-pic"><img src="images/back_image1.jpg" alt=""></div>
+      </div>
+      <!-- UL elements -->
```
Comment (arslanbisharat) [inline index.html:444]: remove comments that are not necessary

File: index.html [hunk 10]
```diff
@@ -4,12 +4,1019 @@
...
+          </li>
+
+          <li>
+            How very tiny technologies are helping tackle the global pandemic
+          </li>
+        </ul>
+      </div>
+    </div>
 
+     <!-- section for single item with flex -->
```
Comment (arslanbisharat) [inline index.html:470]: remove comments that are not necessary

File: index.html [hunk 11]
```diff
@@ -4,12 +4,1019 @@
...
+      <div class="sec-4-big-pic-third">
+        <p class="headline-header bk mainfont">MOST POPULAR</p>
+        <a href="#" class="textstyle-plain">
+          <p class="pic-headline mainfont font-bold">
+            Zeus Arc GT review: You had me at 'ice bong for vapes'
+          </p>
+        </a>
+        <p class="view-history bk mainfont">Tristian Greene - 2 days ago</p>
+      </div>
+      <!-- second div with flex direction -->
```
Comment (arslanbisharat) [inline index.html:483]: remove comments that are not necessary

File: index.html [hunk 12]
```diff
@@ -4,12 +4,1019 @@
...
+
+          <li>
+            Amazon is making a Fallout TV show with Westworld's creators
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <!-- Second row -->
+    <!-- section for single item with flex -->
```
Comment (arslanbisharat) [inline index.html:525]: remove comments that are not necessary

File: index.html [hunk 13]
```diff
@@ -4,12 +4,1019 @@
...
+        <p class="headline-header bk mainfont">MOST POPULAR</p>
+        <a href="#" class="textstyle-plain">
+          <p class="pic-headline mainfont font-bold">
+            Microsoft once owned a chunk of Apple, here’s what it would have
+            been worth today
+          </p>
+        </a>
+        <p class="view-history bk mainfont">Yaron Yitzhak19 hours ago</p>
+      </div>
+      <!-- second div with flex direction -->
```
Comment (arslanbisharat) [inline index.html:540]: remove comments that are not necessary

File: index.html [hunk 14]
```diff
@@ -4,12 +4,1019 @@
...
+          </li>
+
+          <li>
+            An entrepreneur’s guide to COVID-19 tech news in May
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <!-- section for single item with flex -->
```
Comment (arslanbisharat) [inline index.html:581]: remove comments that are not necessary

File: index.html [hunk 15]
```diff
@@ -4,12 +4,1019 @@
...
+    <div class="section-4-item tn-flex">
+      <h2 class="secondaryfont">TECH <i class="fas fa-chevron-right"></i></h2>
+      <!-- Main pic div -->
```
Comment (arslanbisharat) [inline index.html:584]: remove comments that are not necessary

File: index.html [hunk 16]
```diff
@@ -4,12 +4,1019 @@
...
+        <p class="headline-header bk mainfont">MOST POPULAR</p>
+        <a href="#" class="textstyle-plain">
+          <p class="pic-headline mainfont font-bold">
+            Microsoft once owned a chunk of Apple, here’s what it would have
+            been worth today
+          </p>
+        </a>
+        <p class="view-history bk mainfont">Tunde Kehinde 10 days ago</p>
+      </div>
+      <!-- second div with flex direction -->
```
Comment (arslanbisharat) [inline index.html:595]: remove comments that are not necessary

File: index.html [hunk 17]
```diff
@@ -4,12 +4,1019 @@
...
+                scheme’</strong>
+            </p>
+          </a>
+        </div>
+        <div class="small-pic">
+          <img src="images/back_image1.jpg" alt="">
+        </div>
+      </div>
+      <!-- UL elements -->
+      <div class="ul-news-elements">
```
Comment (arslanbisharat) [inline index.html:611]: remove comments that are not necessary

File: index.html [hunk 18]
```diff
@@ -4,12 +4,1019 @@
...
+          </li>
+
+          <li>
+            An entrepreneur’s guide to COVID-19 tech news in May
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <!-- section for single item with flex -->
```
Comment (arslanbisharat) [inline index.html:636]: remove comments that are not necessary

File: index.html [hunk 19]
```diff
@@ -4,12 +4,1019 @@
...
+            Tesla dumped $5B worth of its stock in just 4 days — and traders hate it
+          </li>
+          <li>
+            Satoshi Nakaboto: ‘Researchers find new way to launder money with Bitcoin’
+          </li>
+        </ul>
+      </div>
+    </div>
+    <!-- Third row -->
+    <!-- section for single item with flex -->
```
Comment (arslanbisharat) [inline index.html:687]: remove comments that are not necessary

File: index.html [hunk 20]
```diff
@@ -4,12 +4,1019 @@
...
+    <div class="section-4-item tn-flex">
+      <h2 class="secondaryfont">Hard Fork <i class="fas fa-chevron-right"></i></h2>
+
+      <!-- Main pic div -->
```
Comment (arslanbisharat) [inline index.html:691]: remove comments that are not necessary

File: index.html [hunk 21]
```diff
@@ -4,12 +4,1019 @@
...
+        <p class="headline-header bk mainfont">MOST POPULAR</p>
+        <a href="#" class="textstyle-plain">
+          <p class="pic-headline mainfont font-bold">
+            Microsoft once owned a chunk of Apple, here’s what it would have
+            been worth today
+          </p>
+        </a>
+        <p class="view-history bk mainfont">Yaron Yitzhak19 hours ago</p>
+      </div>
+      <!-- second div with flex direction -->
```
Comment (arslanbisharat) [inline index.html:702]: remove comments that are not necessary

File: index.html [hunk 22]
```diff
@@ -4,12 +4,1019 @@
...
+          <a href="#" class="textstyle-plain">
+            <p class="mainfont">
+              <strong>Satoshi Nakaboto: ‘Ryanair CEO believes Bitcoin is a ponzi
+                scheme’</strong>
+            </p>
+          </a>
+        </div>
+        <div class="small-pic"><img src="images/back_image1.jpg" alt=""></div>
+      </div>
+      <!-- UL elements -->
```
Comment (arslanbisharat) [inline index.html:715]: remove comments that are not necessary

File: index.html [hunk 23]
```diff
@@ -4,12 +4,1019 @@
...
+          </li>
+
+          <li>
+            Amazon is making a Fallout TV show with Westworld's creators
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <!-- section for single item with flex -->
```
Comment (arslanbisharat) [inline index.html:741]: remove comments that are not necessary

File: index.html [hunk 24]
```diff
@@ -4,12 +4,1019 @@
...
+    <div class="section-4-item tn-flex">
+      <h2 class="secondaryfont">TECH <i class="fas fa-chevron-right"></i></h2>
+      <!-- Main pic div -->
```
Comment (arslanbisharat) [inline index.html:744]: remove comments that are not necessary

File: index.html [hunk 25]
```diff
@@ -4,12 +4,1019 @@
...
+        <p class="headline-header bk mainfont">MOST POPULAR</p>
+        <a href="#" class="textstyle-plain">
+          <p class="pic-headline mainfont font-bold">
+            Microsoft once owned a chunk of Apple, here’s what it would have
+            been worth today
+          </p>
+        </a>
+        <p class="view-history bk mainfont">Yaron Yitzhak19 hours ago</p>
+      </div>
+      <!-- second div with flex direction -->
```
Comment (arslanbisharat) [inline index.html:755]: remove comments that are not necessary

File: index.html [hunk 26]
```diff
@@ -4,12 +4,1019 @@
...
+          </li>
+
+          <li>
+            Amazon is making a Fallout TV show with Westworld's creators
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <!-- section for single item with flex -->
```
Comment (arslanbisharat) [inline index.html:794]: remove comments that are not necessary

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes
I can see you worked hard on this project but I am sorry to inform you that I cannot Approve this project because it is not completed yet.😟
_It's okay. We are here to learn. Don't get discouraged. Work more hard._
**Please have a look at the suggestions to get more understanding. 👀**
- [ ] Make your PR more Descriptive by giving more information about your project. 

- [ ] Please resolve all validation errors. You can find those by posting link of index.html into [this link](https://validator.w3.org/) 
validation

- [ ] Remove this bug on small size objects. 
chrome-capture (9)

- [x]  Please remove this space
chrome-capture (10)

- [x] why there is a logo on large screen
chrome-capture (11)

- [x] logo and text is not align as it should be
chrome-capture (12)

- [x] extra space is present here please remove it
chrome-capture (13)

- [x] Remove all comments that are not necessary 

- [x]  Align logo on your footer correctly. 
chrome-capture (8)



[Optional] You have deployed your app on netlify. That's amazing but try to use Github resources like [this](https://raw.githack.com/).  


**Please make the changes and request re-review for the given project**
Stay safe😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Comment by tnyandoro** [comment]
Changes have been made please review the work

**Review by teekaytech — COMMENTED** [review]
# Status:  Changes required :recycle: 

**Hi @tnyandoro, you've really done a lot in this project and I'm happy to tell you that you are almost there :muscle:**

**However, some changes are required which you will have to fix, after which you can request for another review:**

- [ ] This bug still persists on small screen sizes (space to the right). Kindly fix it.



- [ ] Your navigation bar still presents on small screen sizes. 


See the image below from the main website and compare it with yours:


* * *
Kindly make those changes then request another review.

**Taofeek Olalere**
*__Slack (@taofeekolalere) or GitHub (@teekaytech).__*

**Happy coding. :man_technologist:**

**Comment by tnyandoro** [comment]
The navigation bug has been resolved and the extra space on the page has been removed. All the change requests have been resolved.
Kindly review

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
### The [clone](https://zen-feynman-250612.netlify.app/) shows a good understanding and usage of Bootstrap. :100:
##
- The cloned page is rendering as it should and meets all the project specifications.
- CSS is clean. No linter errors at this point and no bad practices used as well.
- The markup as well has no errors nor warnings to show. Good job with that.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @tnyandoro! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**If you have any questions, let me know on slack at any time. I will be happy to help.**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by tnyandoro** [comment]
Merge

**Pull request merged**